Typo fixes (#13113)
docker/docs: language/python/build-images.md

when parsing the Dockerfile, and allows older Docker versions with BuildKit enabled
to upgrade the parser before starting the build. [Parser directives](/engine/reference/builder/#parser-directives)
must appear before any other comment, whitespace, or Dockerfile instruction in

> your Dockerfile, so it should be the first line in Dockerfiles.

Reviewer: ```suggestion
your Dockerfile. Therefore, `syntax` should be the first line in Dockerfiles.
```
